src/components/pagination.js

```diff
@@ -0,0 +1,32 @@
+// outsource dependencies
+import React from 'react';
+import PropTypes from 'prop-types';
+import ReactPaginate from 'react-paginate';
+// local dependencies
+
+function PaginationPage ({ handlePaginationPage, totalPages, page, disabled }) {
+  return <ReactPaginate
+    nextLabel={'next'}
+    forcePage={page}
+    initialPage={page}
+    pageCount={totalPages}
+    previousLabel={'previous'}
```

Comment:
```suggestion
    previousLabel="previous"
```
Why do you wrap it in {}?